### ISSUE TITLE
Firefox - DOMException: The operation is insecure.

### ISSUE THREAD

**Issue by niki-sp** [kkinder/puepy#3]
when testing https://pyscript.com/@kkinder/puepy-tutorial/latest with Firefox / Ubuntu 24.04 i got:
```
DOMException: The operation is insecure.
    lookup_attr micropython.mjs:1
    invoke_iii micropython.mjs:1
    invoke_iiiii micropython.mjs:1
    invoke_ii micropython.mjs:1
    createExportWrapper micropython.mjs:1
    ccall micropython.mjs:1
    runPython micropython.mjs:1
    Lt _python.js:29
    ht/e[t]< utils.js:66
    xn/e[i]< hooks.js:65
    run utils.js:53
    onReady core.js:220
_io.js:40:21
```

**Comment by kkinder** [comment]
At the moment I'm not seeing that with current examples. Are you still?

**Comment by niki-sp** [comment]
Yes examples have same error, but buttons '-' and '+' on https://puepy.dev/ work

**Comment by kkinder** [comment]
@niki-sp Maybe I'm missing something. I have Firefox open to here: https://kkinder.pyscriptapps.com/puepy-tutorial/latest/tutorial/03_counter/index.html

The only output I see in the Firefox console is this:

```
[pyscript/main] PyScript Ready
```

I'm wondering if maybe there's an extension at play, or maybe I'm not recreating the issue correctly?

**Comment by niki-sp** [comment]
Your link https://kkinder.pyscriptapps.com/puepy-tutorial/latest/tutorial/03_counter/index.html works. The problem is on the other link https://pyscript.com/@kkinder/puepy-tutorial/latest
When i click here (Example 3)
![image](https://github.com/kkinder/puepy/assets/17176395/2dba1b5c-9c1d-4879-a099-8eb80b641063)
I get
![image](https://github.com/kkinder/puepy/assets/17176395/ed70bccc-dffc-474d-9c94-59896b7b7a8e)

**Comment by niki-sp** [comment]
IMO it's not very important now, as solution exists

**Comment by kkinder** [comment]
I see. This is likely an upstream issue related to PyScript.com; though I'd have to tinker to be sure. I'll open an issue with them if I can figure out some details.
